USER_MANUAL.md

```diff
@@ -449,9 +449,9 @@ The late connecing client has validated the local state before sending the "Pass
 ##### Player disconnect gym
 * Demonstrates that:
   * Players are cleaned up correctly when they disconnect.
-* NOTE: This gym can only be run manually (This can be automated once the test frameworks supports client disconnects [UNR-529](https://improbableio.atlassian.net/browse/UNR-529)).
+* NOTE: This gym can only be run manually or automatically.
 * Pre-test steps:
-  * In the Unreal Editor's Content Browser, locate `Content/Maps/PlayerDisconnectGym` and double click to open it.
+  * In the Unreal Editor's Content Browser, locate `Content/Maps/PlayerDisconnectGym1` and double click to open it.
```

Comment:
what is the 2 for?

Reply:
When you run the tests manually you can run both tests from the same map but as the automated testing does not re-load the map between tests, the clients do not get re-started so you cannot run two automated disconnect tests on the same map. So I duplicated the map to use for automated testing.

Reply:
I've consolidated this to one map and one test as both tests ultimately tested the same underlying functionality of returning the client to the main menu.